### PR TITLE
remove template from cf-860d

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/cf-860D.mdx [hunk 1]
```diff
@@ -29,233 +29,45 @@ connecting `pre` and `x` is left unpaired and `0` otherwise.
...
+		int a, b;
+		cin >> a >> b;
+		adj[a].push_back(b);
+		adj[b].push_back(a);
 	}
-	dfs(1,-1);
+	dfs(1, -1);
 	cout << ans.size() << "\n";
-	for (auto a: ans) cout << a[0] << " " << a[1] << " " << a[2] << "\n";
+	for (auto a : ans) {
```
Comment (SansPapyrus683) [inline solutions/cf-860D.mdx:68]:
```suggestion
	for (const vector<int>& a : ans) {
```

**Comment by SansPapyrus683** [comment]
bump!

**Review by SansPapyrus683 — APPROVED** [review]
can u test again

**Comment by 876pol** [comment]
yeah it works

**Pull request merged**